synthea/README.md

```diff
@@ -0,0 +1,75 @@
+4. Make sure all profiles are applied to the correct exported resource.
+4. For each must support:
+    - Check the resource's [qicore 4.1.1](https://hl7.org/fhir/us/qicore/STU4.1.1/) profile Snapshot Table to check if the mustSupport is also required by the profile (minimum cardinality 1). If not, it may be initially ignored.
+    - If required, check if synthea already exports it by looking in the [FhirR4.java](https://github.com/synthetichealth/synthea/blob/master/src/main/java/org/mitre/synthea/export/FhirR4.java) file. You can look for a string like `Condition()` for an example of how the Fhir object is created and what fields are set on it.
+    - If Synthea doesn't export it, check if the existing mapping file already adds it through a mapping.
+    - If not, fill from something logical that synthea does export (if something logical exists). Understanding what values are logical may involve looking at the FHIR specification and/or the measure's CQL for how the must support field is used in the measure logic.
+    - If a logical field doesn't exist, fill from a chosen pre-set value or choose randomly from a limited set of values. 
+
+## CMS130 Mapping Example
+
+Below is the example list of resources and must support elements collated from elm-parser-for-ecqms data requirement calculation. Unexported values are marked with a `*` with notes about how their export was resolved in the `qicore_130.yaml` mapping file.
+
+- Condition:
+  - "code",
+  - "clinicalStatus",
+  - "onset",
+  - "abatement"
+- Coverage: * exported as a contained resource only -> ignored since Coverage is only used for SDEs
+  - "type",
+  - "period"
+- DeviceRequest: * not exported -> created based on existing exported Device resources
+  - "code",
+  - "authoredOn",
+  - "status",
+  - "intent",
+  - "modifierExtension", * not exported -> ignored since modifierExtension is only used for doNotPerform
+  - "modifierExtension.url",
+  - "modifierExtension.value"
+- Encounter:
+  - "status",
+  - "type",
+  - "period",
+  - "diagnosis",*
+   - "diagnosis.condition", * not exported -> set using a js function that finds conditions that reference this encounter and creates a reference to that condition
+  - "hospitalization"
+- MedicationRequest:
+  - "medication",
+  - "doNotPerform", * not exported -> set to false because all resources exported from Synthea are logically performed
+  - "status",
+  - "intent",
+  - "dosageInstruction",
+  - "dispenseRequest", * not exported -> set from the `authoredOn` field because the measure logic coalesces `dispenseRequest.validityPeriod.start` with `authoredOn`
+  - "authoredOn"
+- Observation:
+  - "code",
+  - "value",
+  - "effective",
```

Comment:
Same sort of question as above- do we care whether Synthea does or doesn't export all of these choice types? Do we want to indicate they are choice types?